include/capypdf.hpp

```diff
@@ -122,20 +129,51 @@ class GraphicsState : public CapyC<CapyPDF_GraphicsState> {
         _d.reset(gs);
     }
 
+    void set_BM(CapyPDF_Blend_Mode value) { CAPY_CPP_CHECK(capy_graphics_state_set_BM(*this, value)); }
     void set_CA(double value) { CAPY_CPP_CHECK(capy_graphics_state_set_CA(*this, value)); }
+    void set_ca(double value) { CAPY_CPP_CHECK(capy_graphics_state_set_ca(*this, value)); }
 };
 
 class DrawContext : public CapyC<CapyPDF_DrawContext> {
     friend class Generator;
 
 public:
+    void cmd_b() { CAPY_CPP_CHECK(capy_dc_cmd_b(*this)); }
+    void cmd_B() { CAPY_CPP_CHECK(capy_dc_cmd_B(*this)); }
+    void cmd_Bstar() { CAPY_CPP_CHECK(capy_dc_cmd_Bstar(*this)); }
+    void cmd_bstar() { CAPY_CPP_CHECK(capy_dc_cmd_bstar(*this)); }
```

Comment:
The case order is different here. Lowercase versions should come before upper case ones for consistency.